src/internal/utils/bucketHelper.ts

```diff
@@ -197,6 +205,10 @@ export function isAttributeFilter(filter: IBucketFilter): filter is IAttributeFi
     return "attribute" in filter && filter.attribute !== DATE_DATASET_ATTRIBUTE;
 }
 
+export function isMeasureValueFilter(filter: IBucketFilter): filter is IMeasureValueFilter {
```

Comment:
A unit test is missing for this utility. Also, will it survive `null` and `undefined` input?

Reply:
I've added robust testing. BTW: it found a lot of issues and so enforced a much better implementation.